### ISSUE TITLE
Auto slide attribute  

### ISSUE THREAD

**Issue by michaelsam94** [vejei/ViewPagerIndicator#3]
Hello vejei I need to ask you if there is possiblity to add auto slide feature like will be two attributes one is boolean is auto slide or not and other for duration in milliseconds of sliding time interval

**Comment by vejei** [comment]
You don't need to slide automatically, because the indicator works with `ViewPager`(or `ViewPager2`). When the `ViewPager` slides to the next page, the indicator will slide to the next item. Maybe you can make the `ViewPager` slide automatically, and the indicator slides automatically. For the reasons mentioned above, the second attribute is naturally unnecessary.

**Comment by michaelsam94** [comment]
yes I need to make viewpager slide automatically which I achieved with timer task but this timer task could be added to library so that user of library can tell that if they need this feature or not and if they need this feature they can provide duration of slide interval in milleseconds

**Comment by vejei** [comment]
So you mean, add a attribute that allows the indicator to slide automatically, so that the `ViewPager` slides automatically when the indicator slides automatically, right?

That would be too weird. The indicator should follow the `ViewPager`, it slides only when the `ViewPager` slides.

**Comment by michaelsam94** [comment]
ok I got it view pager sliding is responsibility of view pager itself or the component holding the view pager
